content/docs/tutorials/get-started/index.md

```diff
@@ -1,51 +1,59 @@
+### ⚙️ Expand to prepare the project
+
+In expandable sections that start with the ⚙️ emoji, we'll be providing more
+information for those trying to run the commands. It's up to you to pick the
+best way to read the material - read the text (skip sections like this, and it
+should be enough to understand the idea of DVC), or try to run them and get the
+fist hand experience.
+
+We'll be building an NLP project from scratch together. The end result is
+published on [Github](https://github.com/iterative/example-get-started).
+
+Let's start with `git init`:
+
+```dvc
+$ mkdir example-get-started
+$ cd example-get-started
+$ git init
+```
+
+</details>
 
 ```dvc
 $ dvc init
-$ ls .dvc/
-config  plots/  tmp/
 ```
 
-DVC is typically initialized on top of Git, which is needed for the
-[versioning](/doc/tutorials/get-started/data-versioning) features. The `.dvc/`
-directory is automatically staged with Git by `dvc init`, so it can be committed
-right away:
+A few [directories and files](<(/doc/user-guide/dvc-files-and-directories)>) are
+created that should be added into Git:
 
 ```dvc
 $ git status
 Changes to be committed:
         new file:   .dvc/.gitignore
         new file:   .dvc/config
         ...
-$ git commit -m "Initialize DVC repo"
+$ git commit -m "Initialize DVC"
 ```
 
-## What's ahead?
-
-DVC functionality can be split into layers. Each one can be used independently,
-but together they form a robust framework to capture and navigate machine
-learning development.
+DVC functionality can be split into layers and we'll explore them one by one in
+the next few sections:
 
-- [Data **versioning**](/doc/tutorials/get-started/data-versioning) is the basic
-  foundation for storing and sharing **evolving datasets** and ML models. We
-  work on a regular Git workflow, without storing **large files** with Git.
+- [**Data management**](/doc/tutorials/get-started/data-pipelines) is the core
+  part of DVC for large files, datasets, ML models versioning and efficient
+  sharing. We'll show how to use a regular Git workflow, without storing large
+  files with Git. Think "Git for data".
 
-- [Data **pipelines**](/doc/tutorials/get-started/data-pipelines) let you
-  register data modeling **workflows** that can be managed and **reproduced**
-  easily by you or others.
+- [**Data pipelines**](/doc/tutorials/get-started/data-pipelines) desribe how
+  model or some data artifact are built with an efficient way to reproduce them.
+  Think "Makefiles done right" for data and ML projects.
 
-- [**Experiments**](/doc/tutorials/get-started/experiments) are a natural part
-  of data science, or any R&D process. DVC provides special tools to define,
-  manage, tune, and **compare them** through _parameters_ and _metrics_.
+- [**Experiments**](/doc/tutorials/get-started/experiments) attach parameters,
+  metrics, plots. You can capture and navigate experiments not leaving Git.
+  Think "Git for ML".
```

Comment:
```suggestion
- [**Experiments**](/doc/tutorials/get-started/experiments) attach parameters,
  metrics, and plots. You can capture and navigate experiments without leaving Git.
  Think "Git for ML".
```
> Needs rewrap.

But I don't get this last part about not leaving Git or "Git for ML". Sounds redundant with bullet one's "Git for data". Maybe this bullet doesn't need the last sentence.